packages/hydrogen/src/cart/cart-test-helper.ts

```diff
@@ -0,0 +1,79 @@
+import {CachingStrategy} from '../cache/strategies';
+import type {ExecutionArgs} from 'graphql';
+import {Storefront} from '../storefront';
+import {CacheNone} from '../cache/strategies';
+
+export const CART_ID = 'gid://shopify/Cart/c1-123';
+export const NEW_CART_ID = 'c1-new-cart-id';
+
+function storefrontQuery(
+  query: string,
+  payload?: {
+    variables?: ExecutionArgs['variableValues'];
+    cache?: CachingStrategy;
+  },
+) {
+  return Promise.resolve({
+    cart: {
+      id: payload?.variables?.cartId,
+      query,
+    },
+  });
+}
+
+function storefrontMutate(
+  query: string,
+  payload?: {
+    variables?: ExecutionArgs['variableValues'];
+  },
+) {
+  let cartId = payload?.variables?.cartId as string;
+  let keyWrapper: string = 'error';
+  if (/mutation CartCreate/.test(query)) {
```

Comment:
Would `keyWrapper = query.match(/mutation\s+([^\(\s]+)/i)?.[1]` work here instead of the if/else? (and maybe lowercase the first letter of the match).